tensorboard/webapp/feature_flag/store/feature_flag_selectors.ts

```diff
@@ -36,25 +37,32 @@ export const getIsFeatureFlagsLoaded = createSelector(
 
 export const getFeatureFlags = createSelector(
   selectFeatureFlagState,
-  (state) => {
-    return state.features;
+  (state: FeatureFlagState): FeatureFlags => {
+    return {...state.defaultFlags, ...state.flagOverrides};
   }
 );
 
-export const getEnabledExperimentalPlugins = createSelector(
+export const getOverridenFeatureFlags = createSelector(
   selectFeatureFlagState,
+  (state: FeatureFlagState): Partial<FeatureFlags> => {
+    return state.flagOverrides;
+  }
+);
+
+export const getEnabledExperimentalPlugins = createSelector(
+  getFeatureFlags,
   (state) => {
```

Comment:
now that the parameter is no longer `FeatureFlagState`, can we rename these `state` params to `flags`, here and below?